the navigation and authors config to be under the '/config' namespace

SaxenaShiv opened the pull request:
Hi, I have added navigation.ts and authors.ts under ./config folder and added the sample code.

 - [x] Created navigation.ts file and added the documentation.ts code from navs folder
- [x] Created authors type (Optional)
- [x] Created authors.ts file
- [x] Deleted navs folder

SaxenaShiv committed:
authors and navs under `./config`

SaxenaShiv committed:
Merge branch 'elegantframework:v2.3' into v2.3

SaxenaShiv committed:
Merge branch 'v2.3' into v2.3

SaxenaShiv committed:
documentation problem fix

SaxenaShiv committed:
Merge branch 'v2.3' of https://github.com/Shiv-2712/elegant-cli into v2.3

SaxenaShiv committed:
arrow icon beside external link title

Bowens20832 committed:
Merge branch 'v2.3' into v2.3

Bowens20832 committed:
Revert "arrow icon beside external link title"

This reverts commit 88084d1ee94ffc8b85f635eaf81a01f214e8c31c.

Bowens20832 reviewed (approved):
I think you merged you code from #81 into this branch as well.

I have moved that code into it's own feature branch and will open a PR and review it for you 😄 

Thank you for the code contribution!

Bowens20832 committed:
Update release-notes.mdx

SaxenaShiv committed:
Merge branch 'v2.3' into v2.3

SaxenaShiv committed:
Merge branch 'v2.3' into v2.3

Bowens20832 reviewed (approved):
Looks good to me : )

Bowens20832 committed:
Merge branch 'v2.3' into pr/107

Bowens20832 committed:
Update release-notes.mdx

Bowens20832 committed:
Update for production use.

- Updated Authors.ts to match existing naming style convention.
- Updated Navigation.ts to match existing naming style convention.
- Migrated and deleted depreacted authors.js file.
- Wrote Jest test for new Authors.ts file.
- Updated package-lock.json file.

Ready for production use. Thanks @Shiv-2712 for the contribution! :smile:

Co-Authored-By: Shiv Kumar Saxena <74370135+Shiv-2712@users.noreply.github.com>

Bowens20832 committed:
2.3.1-innovators.11
